src/components/TransactionTable/TransactionTable.js

```diff
@@ -0,0 +1,75 @@
+import React, { PureComponent } from 'react';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import { Table } from '../index';
+import { connectTheme } from '../../utils';
+
+import { TxnManager, TxnManagerOptions } from '@bpanel/bpanel-utils';
+
+import TransactionView from './TransactionView';
+
+class TransactionTable extends PureComponent {
+  constructor() {
+    super();
+    // TODO: figure out way to accept
+    // certain props and render those rows
```

Comment:
Rows are just transactions right? Are you thinking of a way to filter for certain transactions based on props?

Reply:
Rows are transactions, the consumer can filter txs before passing them into the table
The idea is to pass a map between `uxtx.toJSON -> columns` to be displayed in the table